applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -173,6 +248,54 @@ static uint32_t get_and_incr_seq_num(const struct bt_audio_stream *stream)
 	return 0;
 }
 
+static int update_sink_stream_qos(uint32_t pres_delay_us)
+{
+	int ret;
+
+	lc3_preset_sink.qos.pd = pres_delay_us + 10;
+
+	for (int i = 0; i < ARRAY_SIZE(headsets); i++) {
+		LOG_INF("Headset %d, connection", i);
+
+		if (headsets[i].headset_conn != NULL && headsets[i].sink_stream.ep != NULL) {
```

Comment:
To get less indentations, I suggest: 
```suggestion
		if (headsets[i].headset_conn == NULL || headsets[i].sink_stream.ep == NULL) {return -EINVAL }
```

Reply:
This has all changed and is no longer relevant.